docs/maps/connect-to-ems.asciidoc

```diff
@@ -13,7 +13,16 @@ EMS requests are made to the following domains:
 * vector.maps.elastic.co
 
 **Elastic Maps** makes requests directly from the browser to EMS.
-To proxy EMS requests through the Kibana server, set `map.proxyElasticMapsServiceInMaps` to `true` in your <<settings, kibana.yml>> file.
+
+[float]
+=== Connecting to Elastic Maps Service from an internal network
```

Comment:
Let's use this form for the two headings in this doc:

Connect to Elastic Maps Service from an internal network
Disable Elastic Maps Service